src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsUnitTest.java

```diff
@@ -226,19 +227,20 @@ public void testReadWithNsRefIndexInDeletion() throws FileNotFoundException {
     }
 
     @Test
-    public void testReadWithNsRefAfterDeletion() throws FileNotFoundException {
-
-        final IndexedFastaSequenceFile seq = new CachingIndexedFastaSequenceFile(IOUtils.getPath(exampleReference));
-        final SAMFileHeader header = ArtificialReadUtils.createArtificialSamHeader(seq.getSequenceDictionary());
-        final int readLength = 76;
-
-        final GATKRead read = ArtificialReadUtils.createArtificialRead(header, "myRead", 0, 8975, readLength);
-        read.setBases(Utils.dupBytes((byte) 'A', readLength));
-        read.setBaseQualities(Utils.dupBytes((byte)30, readLength));
-        read.setCigar("3M414N1D73M");
-
-        final int result = ReadUtils.getReadCoordinateForReferenceCoordinateUpToEndOfRead(read, 9393, ReadUtils.ClippingTail.LEFT_TAIL);
-        Assert.assertEquals(result, 3);
+    public void testReadWithNsRefAfterDeletion() throws IOException {
+        try(final ReferenceSequenceFile seq = new CachingIndexedFastaSequenceFile(IOUtils.getPath(exampleReference))) {
+            final SAMFileHeader header = ArtificialReadUtils.createArtificialSamHeader(seq.getSequenceDictionary());
+            final int readLength = 76;
+
+            final GATKRead read = ArtificialReadUtils.createArtificialRead(header, "myRead", 0, 8975, readLength);
+            read.setBases(Utils.dupBytes((byte) 'A', readLength));
+            read.setBaseQualities(Utils.dupBytes((byte) 30, readLength));
+            read.setCigar("3M414N1D73M");
+
+            final int result = ReadUtils.getReadCoordinateForReferenceCoordinateUpToEndOfRead(read, 9393,
+                                                                                              ReadUtils.ClippingTail.LEFT_TAIL);
+            Assert.assertEquals(result, 3);
+        }
     }
 
     @DataProvider(name = "HasWellDefinedFragmentSizeData")
```

Comment:
Can you add at least one integration test that runs a walker with a `.fasta.gz` specified for the `-R` argument, to prove that walker support is hooked up? Should pick a tool that uses the reference in some easy-to-verify way, such as `ExampleReadWalkerWithReference`

Reply:
I've added a second case to ExampleReadWalkerWithReferenceIntegrationTest that uses the gzipped hg19minireference